db/migrate/20151214060428_add_css_to_companies.rb

```diff
@@ -0,0 +1,5 @@
+class AddCssToCompanies < ActiveRecord::Migration
+  def change
+    add_column :companies, :css, :string
```

Comment:
@arafatm NO, the css file is stored in `app/assets/stylesheets` directory... this attribute just stores the name of the file which links to the original file on asset directory